charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -288,6 +294,9 @@ scrape_configs:
     target_label: kubernetes_name
   - source_labels: [__meta_kubernetes_ingress_path]
     target_label: path
+  - target_label: region_probed_from
+    replacement: {{ . }}
+{{- end}}
```

Comment:
Isn't that already clear from the proper URL? It's also another drop of special SAP sauce...

Reply:
Good point. By default the prober URI is dropped, but I will use a more generic no-SAP way to keep the information.